ophyd/sim.py

```diff
@@ -1240,6 +1257,10 @@ def hw(save_path=None):
     # area detector that directly stores image data in Event
     direct_img = SynSignal(func=lambda: np.array(np.ones((10, 10))),
                            name='img', labels={'detectors'})
+
+    direct_img_list = SynSignal(func=lambda: [[1] * 10] * 10,
+                                name='img', labels={'detectors'})
```

Comment:
```suggestion
                                name='img_list', labels={'detectors'})
```

Reply:
@mrakitin I think we should have a different name for this detector to avoid confusion (see above)

Reply:
Good point, @awalter-bnl, I don't mind to change it, however I followed the pattern as 2 detectors had the same name attribure (see the `img` det below). I think we then need to make them all unique.

Reply:
@mrakitin you are correct, I made that suggestion in your PR to `suitcase-utils` :-)